### PR TITLE
Correct word usage

### LINKED ISSUE #26461: webpage or web page
### MDN URL

https://developer.mozilla.org/en-US/docs/Learn/Common_questions/Web_mechanics/Pages_sites_servers_and_search_engines

### What specific section or headline is this issue about?

_No response_

### What information was incorrect, unhelpful, or incomplete?

I found this page trying to determine if it is correct to say "webpage" or "web page".  But this page has both!  Would it not be proper to be consistent?  I also see an inconsistency in "website" vs "web site".

### What did you expect to see?

consistent presentation

### Do you have any supporting links, references, or citations?

_No response_

### Do you have anything more you want to share?

_No response_

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/learn/common_questions/web_mechanics/pages_sites_servers_and_search_engines`
* MDN URL: https://developer.mozilla.org/en-US/docs/Learn/Common_questions/Web_mechanics/Pages_sites_servers_and_search_engines
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/learn/common_questions/web_mechanics/pages_sites_servers_and_search_engines/index.md
* Last commit: https://github.com/mdn/content/commit/8e2641ebe076ab89299c77a51ece882de4ba5efb
* Document last modified: 2023-02-23T23:55:46.000Z

</details>

### PR DESCRIPTION
**Pull request by dawei-wang**
Correct "web site" and "web page" to single words to improve consistency.

Fix mdn/content#26461

### CODE REVIEW

**Review by wbamberg — COMMENTED** [review]
Thanks for your PR, @dawei-wang !

I think this is not clear though. According to the Google style guide, "web pages" is preferred (https://developers.google.com/style/word-list#web-page). I've asked around a bit and to the extent that people have expressed a preference, "web page" is preferred (which is also my preference, FWIW).

Perhaps you could make a PR to update https://developer.mozilla.org/en-US/docs/MDN/Writing_guidelines/Writing_style_guide to add a "selective word list", where we could make these choices and get feedback? Then once we have that, we can update the content?

I think it would be best as an H2 after "Writing style".

(Alternatively, we could simply refer to another word list, like https://developers.google.com/style/word-list, and save us having to maintain our own. At least, we could say something like, this is a selective list and for any words not listed here, refer to https://developers.google.com/style/word-list.)

What do you think?

**Comment by dawei-wang** [comment]
> (Alternatively, we could simply refer to another word list, like https://developers.google.com/style/word-list, and save us having to maintain our own. At least, we could say something like, this is a selective list and for any words not listed here, refer to https://developers.google.com/style/word-list.)
> 
I like the idea of referring to an alternative word list. My original thought was to refer to the [Microsoft Style Guide](https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/w/web-world-wide-web-www) because we already point users to this resource for guidelines we do not cover. Microsoft seems to prefer website and webpage over web site and web page FWIW.

**Comment by wbamberg** [comment]
> > (Alternatively, we could simply refer to another word list, like https://developers.google.com/style/word-list, and save us having to maintain our own. At least, we could say something like, this is a selective list and for any words not listed here, refer to https://developers.google.com/style/word-list.)
> 
> @wbamberg I like the idea of referring to another word list. My original thought was to refer to the [Microsoft Style Guide](https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/w/web-world-wide-web-www) because we already point users to this resource for guidelines we do not cover. 

Oh, you meAN HERE: https://developer.mozilla.org/en-US/docs/MDN/Writing_guidelines/Writing_style_guide#preferred_style_guides ? I hadn't even noticed that before...

> Microsoft seems to prefer website and webpage over web site and web page FWIW.

Yeah I know. Everyone seems to agree on website but not on web[ ]page. Let's start a discussion.

**Comment by dawei-wang** [comment]
> Oh, you mean here: https://developer.mozilla.org/en-US/docs/MDN/Writing_guidelines/Writing_style_guide#preferred_style_guides ? I hadn't even noticed that before...
> 
Exactly

**Comment by wbamberg** [comment]
I've started a discussion here: https://github.com/orgs/mdn/discussions/372. Let's see what happens.

**Comment by wbamberg** [comment]
All right, so [if I understand correctly](https://github.com/orgs/mdn/discussions/372#discussioncomment-5821240), we should be using "website" and "web page". Does that sound good to you?

**Comment by dawei-wang** [comment]
Sounds good to me.

File: files/en-us/learn/common_questions/web_mechanics/pages_sites_servers_and_search_engines/index.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ slug: Learn/Common_questions/Web_mechanics/Pages_sites_servers_and_search_engine
 
 {{QuicklinksWithSubPages("Learn/Common_questions")}}
 
-In this article, we describe various web-related concepts: web pages, websites, web servers, and search engines. These terms are often confused by newcomers to the web or are incorrectly used. Let's learn what they each mean!
+In this article, we describe various web-related concepts: webpages, websites, web servers, and search engines. These terms are often confused by newcomers to the web or are incorrectly used. Let's learn what they each mean!
```
Comment (dawei-wang) [inline files/en-us/learn/common_questions/web_mechanics/pages_sites_servers_and_search_engines/index.md:8]:
```suggestion
In this article, we describe various web-related concepts: web pages, websites, web servers, and search engines. These terms are often confused by newcomers to the web or are incorrectly used. Let's learn what they each mean!
```

File: files/en-us/learn/common_questions/web_mechanics/pages_sites_servers_and_search_engines/index.md [hunk 2]
```diff
@@ -34,7 +34,7 @@ As with any area of knowledge, the web comes with a lot of jargon. Don't worry.
 
 We'll cover these terms and technologies in more detail as we explore further, but these quick definitions will be a great start for you:
 
-- web page
+- webpage
```
Comment (dawei-wang) [inline files/en-us/learn/common_questions/web_mechanics/pages_sites_servers_and_search_engines/index.md:37]:
```suggestion
- web page
```

File: files/en-us/learn/common_questions/web_mechanics/pages_sites_servers_and_search_engines/index.md [hunk 3]
```diff
@@ -64,9 +64,9 @@ _There is no active learning available yet. [Please, consider contributing](/en-
 
 So, let's dig deeper into how those four terms are related and why they are sometimes confused with each other.
 
-### Web page
+### Webpage
```
Comment (dawei-wang) [inline files/en-us/learn/common_questions/web_mechanics/pages_sites_servers_and_search_engines/index.md:67]:
```suggestion
### Web page
```

File: files/en-us/learn/common_questions/web_mechanics/pages_sites_servers_and_search_engines/index.md [hunk 4]
```diff
@@ -64,9 +64,9 @@ _There is no active learning available yet. [Please, consider contributing](/en-
...
 
-A **web page** is a simple document displayable by a {{Glossary("browser")}}. Such documents are written in the {{Glossary("HTML")}} language (which we look into in more detail in [other articles](/en-US/docs/Web/HTML)). A web page can embed a variety of different types of resources such as:
+A **webpage** is a simple document displayable by a {{Glossary("browser")}}. Such documents are written in the {{Glossary("HTML")}} language (which we look into in more detail in [other articles](/en-US/docs/Web/HTML)). A web page can embed a variety of different types of resources such as:
```
Comment (dawei-wang) [inline files/en-us/learn/common_questions/web_mechanics/pages_sites_servers_and_search_engines/index.md:69]:
```suggestion
A **web page** is a simple document displayable by a {{Glossary("browser")}}. Such documents are written in the {{Glossary("HTML")}} language (which we look into in more detail in [other articles](/en-US/docs/Web/HTML)). A web page can embed a variety of different types of resources such as:
```

File: files/en-us/learn/common_questions/web_mechanics/pages_sites_servers_and_search_engines/index.md [hunk 5]
```diff
@@ -78,15 +78,15 @@ All web pages available on the web are reachable through a unique address. To ac
 
 ![Example of a web page address in the browser address bar](web-page.jpg)
 
-### Web site
+### Website
 
-A _website_ is a collection of linked web pages (plus their associated resources) that share a unique domain name. Each web page of a given website provides explicit links—most of the time in the form of clickable portions of text—that allow the user to move from one page of the website to another.
+A _website_ is a collection of linked webpages (plus their associated resources) that share a unique domain name. Each webpage of a given website provides explicit links—most of the time in the form of clickable portions of text—that allow the user to move from one page of the website to another.
```
Comment (dawei-wang) [inline files/en-us/learn/common_questions/web_mechanics/pages_sites_servers_and_search_engines/index.md:83]:
```suggestion
A _website_ is a collection of linked web pages (plus their associated resources) that share a unique domain name. Each web page of a given website provides explicit links—most of the time in the form of clickable portions of text—that allow the user to move from one page of the website to another.
```

File: files/en-us/learn/common_questions/web_mechanics/pages_sites_servers_and_search_engines/index.md [hunk 6]
```diff
@@ -78,15 +78,15 @@ All web pages available on the web are reachable through a unique address. To ac
...
 
 To access a website, type its domain name in your browser address bar, and the browser will display the website's main web page, or _homepage_ (casually referred as "the home"):
 
 ![Example of a web site domain name in the browser address bar](web-site.jpg)
 
-The ideas of a _web page_ and a _website_ are especially easy to confuse for a _website_ that contains only one _web page._ Such a website is sometimes called a _single-page website._
+The ideas of a _webpage_ and a _website_ are especially easy to confuse for a _website_ that contains only one _webpage._ Such a website is sometimes called a _single-page website._
```
Comment (dawei-wang) [inline files/en-us/learn/common_questions/web_mechanics/pages_sites_servers_and_search_engines/index.md:89]:
```suggestion
The ideas of a _web page_ and a _website_ are especially easy to confuse for a _website_ that contains only one _webpage._ Such a website is sometimes called a _single-page website._
```

**Review by dawei-wang — COMMENTED** [review]
Correct webpage to two words (match spelling guide).

**Review by dawei-wang — COMMENTED** [review]
Correct webpage to two words (match spelling guide).

File: files/en-us/learn/common_questions/web_mechanics/pages_sites_servers_and_search_engines/index.md [hunk 7]
```diff
@@ -78,15 +78,15 @@ All web pages available on the web are reachable through a unique address. To ac
...
+### Website
 
 A _website_ is a collection of linked web pages (plus their associated resources) that share a unique domain name. Each web page of a given website provides explicit links—most of the time in the form of clickable portions of text—that allow the user to move from one page of the website to another.
 
 To access a website, type its domain name in your browser address bar, and the browser will display the website's main web page, or _homepage_ (casually referred as "the home"):
 
 ![Example of a web site domain name in the browser address bar](web-site.jpg)
 
-The ideas of a _web page_ and a _website_ are especially easy to confuse for a _website_ that contains only one _web page._ Such a website is sometimes called a _single-page website._
+The ideas of a _web page_ and a _website_ are especially easy to confuse for a _website_ that contains only one _webpage._ Such a website is sometimes called a _single-page website._
```
Comment (hamishwillee) [inline files/en-us/learn/common_questions/web_mechanics/pages_sites_servers_and_search_engines/index.md:89]: I don't find this sentence useful - it feels like we're comparing things that are not confusing in order to make them confusing. Also the `webpage` at the end - is that intentionally not spaced?

```suggestion
Note that it also possible to have a  a _single-page website: a site that consists of a single web page which is dynamically updated with new content when needed.
```

File: files/en-us/learn/common_questions/web_mechanics/pages_sites_servers_and_search_engines/index.md [hunk 8]
```diff
@@ -78,15 +78,15 @@ All web pages available on the web are reachable through a unique address. To ac
...
+### Website
 
 A _website_ is a collection of linked web pages (plus their associated resources) that share a unique domain name. Each web page of a given website provides explicit links—most of the time in the form of clickable portions of text—that allow the user to move from one page of the website to another.
 
 To access a website, type its domain name in your browser address bar, and the browser will display the website's main web page, or _homepage_ (casually referred as "the home"):
 
 ![Example of a web site domain name in the browser address bar](web-site.jpg)
 
-The ideas of a _web page_ and a _website_ are especially easy to confuse for a _website_ that contains only one _web page._ Such a website is sometimes called a _single-page website._
+Note that it also possible to have a  a _single-page website_: a site that consists of a single web page which is dynamically updated with new content when needed.
```
Comment (dawei-wang) [inline files/en-us/learn/common_questions/web_mechanics/pages_sites_servers_and_search_engines/index.md:89]:
```suggestion
Note that it also possible to have a _single-page website_: a site that consists of a single web page which is dynamically updated with new content when needed.
```

**Review by dawei-wang — COMMENTED** [review]
delete extra space and repeated word

**Review by teoli2003 — APPROVED** [review]
👍

**Pull request merged**